src/lang/std/sketchcombos.ts

```diff
@@ -110,6 +122,39 @@ function createCallWrapper(
   tag?: Expr,
   valueUsedInTransform?: number
 ): CreatedSketchExprResult {
+  if (Array.isArray(val)) {
```

Comment:
We should use `isArray()` in `src/lib/utils.ts`.

Reply:
`Array.isArray` is used 8 other places in this file, I would like to be consistent with the rest of the file. Follow-up PR could stop using that function.